mod-orders-storage/schemas/piece.json

```diff
@@ -49,6 +45,11 @@
       "description": "Whether or not this is supplementary material",
       "type": "boolean"
     },
+    "receiptDate": {
```

Comment:
Requirements question - should `receivedDate` be less than or equals to `receiptDate`? What happens if `piece` is received after `receiptDate`? `receivedDate` is optional so technically it can be anything.

Also, if possible, please change the description of `receivedDate` to something like `"The date this item is actually received"`